build.gradle

```diff
@@ -41,40 +45,48 @@ dependencyManagement {
 }
 
 dependencies {
+    // Declared
+    compile 'org.codehaus.groovy:groovy-json'
+    compile "org.hibernate:hibernate-ehcache:5.4.0.Final"
+
+    // Grails Generated
+    developmentOnly("org.springframework.boot:spring-boot-devtools")
     compile "org.springframework.boot:spring-boot-starter-logging"
     compile "org.springframework.boot:spring-boot-autoconfigure"
-    compile "org.grails:grails-core"
-    compile "org.springframework.boot:spring-boot-starter-actuator"
     compile "org.springframework.boot:spring-boot-starter-tomcat"
-    compile "org.grails:grails-dependencies"
-    compile "org.grails:grails-web-boot"
-    compile "org.grails.plugins:cache"
-    compile "org.grails.plugins:scaffolding"
+    compile "org.grails:grails-core"
+    compile "org.grails:grails-plugin-i18n"
+    compile "org.grails:grails-plugin-url-mappings"
+    compile "org.grails:grails-plugin-services"
+    compile "org.grails.plugins:async"
+    compile "org.grails.plugins:hibernate5"
+    compile "org.hibernate:hibernate-core:5.4.0.Final"
+    compileOnly "io.micronaut:micronaut-inject-groovy"
     console "org.grails:grails-console"
     profile "org.grails.profiles:plugin"
-    provided "org.grails:grails-plugin-services"
-    provided "org.grails:grails-plugin-domain-class"
-    testCompile "org.grails:grails-plugin-testing"
-    testCompile "org.grails.plugins:geb"
-    testRuntime "org.seleniumhq.selenium:selenium-htmlunit-driver:2.47.1"
-    testRuntime "net.sourceforge.htmlunit:htmlunit:2.18"
-
-    compile "org.grails.plugins:hibernate4"
-    compile "org.hibernate:hibernate-ehcache"
     runtime "com.h2database:h2"
+    runtime "org.apache.tomcat:tomcat-jdbc"
```

Comment:
Is this needed?

Reply:
Without this dependency, the tests fail with the following error:

```text
ERROR org.hibernate.engine.jdbc.spi.SqlExceptionHelper - Table "ASYNCHRONOUS_TASK_DOMAIN" not found; SQL statement:
insert into asynchronous_task_domain (id, internal_task_data, progress, date_created, results, error_code, end_time, last_updated, resolution_code, start_time, name, description, state, current_operation) values (null, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?) [42102-199]

Hibernate operation: could not prepare statement; bad SQL grammar [insert into asynchronous_task_domain (id, internal_task_data, progress, date_created, results, error_code, end_time, last_updated, resolution_code, start_time, name, description, state, current_operation) values (null, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?)]; nested exception is org.h2.jdbc.JdbcSQLSyntaxErrorException: Table "ASYNCHRONOUS_TASK_DOMAIN" not found; SQL statement:
insert into asynchronous_task_domain (id, internal_task_data, progress, date_created, results, error_code, end_time, last_updated, resolution_code, start_time, name, description, state, current_operation) values (null, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?) [42102-199]
org.springframework.jdbc.BadSqlGrammarException: Hibernate operation: could not prepare statement; bad SQL grammar [insert into asynchronous_task_domain (id, internal_task_data, progress, date_created, results, error_code, end_time, last_updated, resolution_code, start_time, name, description, state, current_operation) values (null, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?)]; nested exception is org.h2.jdbc.JdbcSQLSyntaxErrorException: Table "ASYNCHRONOUS_TASK_DOMAIN" not found; SQL statement:
insert into asynchronous_task_domain (id, internal_task_data, progress, date_created, results, error_code, end_time, last_updated, resolution_code, start_time, name, description, state, current_operation) values (null, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?) [42102-199]
```